mlflow/server/handlers.py

```diff
@@ -126,7 +127,7 @@ def _create_run():
         source_type=request_message.source_type,
         source_name=request_message.source_name,
         entry_point_name=request_message.entry_point_name,
-        start_time=request_message.start_time,
+        start_time=request_message.start_time or int(time.time()),
```

Comment:
Thanks for noticing this, I'd hold off on making this change for now / in this PR since we don't set defaults for any of the other fields in our REST API - would you mind raising a GitHub issue to track this? It's probably worth discussing whether we set default values for fields or just raise an exception if a required field is missing.

Reply:
Thanks for creating the issue! Yeah definitely agree it's best to strive for similar behavior across the Python client & REST API & that default/required fields should be handled as you described.

In the case of run creation time it's tricky since the meaning of `start_time` becomes ambiguous if we allow the server to set it (i.e. if a user looks at a run's start time, they'd no longer be sure if the start time was when the run was created on the client or when the request was received by the server, and the two times could be pretty different depending on network strength etc). We have the same problem with the `UpdateRun` endpoint.

For context: one use case we'd imagined for allowing clients to specify run start/end timestamps was that clients could cache run data if the network is unstable & submit run info to the server later. IMO between setting a default start time in the server or validating that start_time is provided by the client, we should lean towards the latter, but would be curious to hear your thoughts. Regardless I know others (like @mparkhe @aarondav) have opinions on this too, so would be great if we could discuss it on #354 & address it in a separate PR :)

Reply:
Sure, we can discuss this there. Thanks again.